frontend/src/components/Shared/footer/Content.jsx

```diff
@@ -96,9 +95,14 @@ const Nav = () => {
     },
     {
       name: 'Tiktok',
-      link: 'https://www.tiktok.com/@sipnplaynycofficial?lang=en',
+      link: 'https://www.tiktok.com/@sipnplaynyc?lang=en',
```

Comment:
@devxMani why did you change the user name ?

Reply:
i think by mistake hogya hoga

Reply:
ok buddy thik karle 😁